app/models/concerns/alegre_v2.rb

```diff
@@ -1,4 +1,41 @@
 require 'active_support/concern'
+class TemporaryProjectMedia
+  attr_accessor :team_id, :id, :url, :type, :media
+  def media
```

Comment:
@DGaffney , since `media` is defined as an `attr_accessor`, this means that methods `media` (getter), `media=` (setter) are created... but then you define a method called `media` (like a custom getter). If you just want the setter, please use `attr_writer` for `:media` instead of `attr_accessor`.